jkube-kit/config/service/src/test/java/org/eclipse/jkube/kit/config/service/openshift/OpenshiftBuildServiceIntegrationTest.java

```diff
@@ -126,8 +132,12 @@ void init(@TempDir Path temporaryFolder) throws Exception {
     jKubeBuildTarArchiver = mockConstruction(JKubeBuildTarArchiver.class, (mock, ctx) ->
         when(mock.createArchive(any(File.class), any(BuildDirs.class), eq(ArchiveCompression.none)))
         .thenReturn(emptyDockerBuildTar));
-
-    resourceConfig = ResourceConfig.builder().namespace("ns1").build();
+    
+    props = new Properties();
+    props.setProperty("buildName", projectName);
```

Comment:
Is this really required? Even if I remove it test passes. 

Is it a leftover from when you were facing test failures https://github.com/eclipse-jkube/jkube/issues/2844#issuecomment-2075857092 ?